docs/resources/Guild.md

```diff
@@ -644,6 +645,24 @@ In guilds with [Membership Screening](https://support.discord.com/hc/en-us/artic
 > warn
 > We are making significant changes to the Membership Screening API specifically related to getting and editing the Membership Screening object. Long story short is that it can be improved. As such, we have removed those documentation. There will **not be** any changes to how pending members work, as outlined above. That behavior will stay the same.
 
+### Incidents Data Object
+
+###### Incidents Data Structure
+
+| Field                  | Type               | Description                            |
+|------------------------|--------------------|----------------------------------------|
+| invites_disabled_until | ?ISO8601 timestamp | when invites get enabled again         |
+| dms_disabled_until     | ?ISO8601 timestamp | when direct messages get enabled again |
```

Comment:
Does this follow the same pattern as user timeouts (null or value in the past if disabled)? Or is it just null if disabled?
Either way, it'd be worth documenting / noting, whether in a similar fashion to the guild member object or another way...

Reply:
> Does this follow the same pattern as user timeouts (null or value in the past if disabled)? Or is it just null if disabled? Either way, it'd be worth documenting / noting, whether in a similar fashion to the guild member object or another way...

Just tested this, it gets set back to `null` after the time expires.

Reply:
If either time runs out => `null`
If both run out, the guild field with the object is `nul`